lib/beaker/host/unix/pkg.rb

```diff
@@ -120,6 +120,34 @@ def install_package(name, cmdline_args = '', version = nil, opts = {})
     end
   end
 
+  # Extract RPM command's proxy options from URL
+  #
+  # @param [Url] url A URL of form http://host:port
+  #
+  def extract_rpm_proxy_options(url)
```

Comment:
Yes, I do consider that dealt with.

As guarding against passing in nil, I consider this a private method; so no one should call it other than the install_package_with_rpm method so I think it is a better balance to keep the code clean and not add logic to handle input that can never be passed in?

Perhaps I should explicitly mark this method private and add @api private as I see has been done in windows/pkg.rb.

Happy to add quotes around url.